index.html

```diff
@@ -37,13 +40,150 @@ <h1 class="supporting-text">I'm a software developer</h1>
+            <li class="tags-li">HTML</li>
+            <li class="tags-li">Bootstrap</li>
+            <li class="tags-li">Ruby</li>
+          </ul>
+        </div>
+        <button class="see-project"> See Project</button>
+      </div>
+      <div class="card">
+        <div class="portrait">
+          <h3 class="profesional-art">Profesional Art Printing Data</h3>
+          <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="tags">
+            <li class="tags-li">HTML</li>
+            <li class="tags-li">Bootstrap</li>
+            <li class="tags-li">Ruby</li>
+          </ul>
+        </div>
+        <button class="see-project"> See Project</button>
+      </div>
+      <div class="card">
+        <div class="portrait">
+          <h3 class="profesional-art">Profesional Art Printing Data</h3>
+          <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="tags">
+            <li class="tags-li">HTML</li>
+            <li class="tags-li">Bootstrap</li>
+            <li class="tags-li">Ruby</li>
+          </ul>
+        </div>
+        <button class="see-project"> See Project</button>
+      </div>
+      <div class="card">
+        <div class="portrait">
+          <h3 class="profesional-art">Profesional Art Printing Data</h3>
+          <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="tags">
+            <li class="tags-li">HTML</li>
+            <li class="tags-li">Bootstrap</li>
+            <li class="tags-li">Ruby</li>
+          </ul>
+        </div>
+        <button class="see-project"> See Project</button>
+      </div>
+      <div class="card">
+        <div class="portrait">
+          <h3 class="profesional-art">Profesional Art Printing Data</h3>
+          <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="tags">
+            <li class="tags-li">HTML</li>
+            <li class="tags-li">Bootstrap</li>
+            <li class="tags-li">Ruby</li>
+          </ul>
+        </div>
+        <button class="see-project"> See Project</button>
+      </div>
 
+    </section>
 
-    </ul>
+  </section>
+  <section class="about-me-section">
+    <div class="about-me">
+      <h3 class="about-title">About me</h3>
+      <p class="description-about">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <button class="resume-button">Get My Resume</button>
+    </div>
```

Comment:
- [ ] Your button is really great, But we need to follow the button styling: `Active`, `Hover`, and `Pressed`, Please add these effects for a better user interface.
![Screenshot 2023-01-14 081956](https://user-images.githubusercontent.com/69077061/212457555-ea44ba6e-653e-48a2-8e98-ae08d40d4ddf.png)